cosmic-client/src/main/webapp/scripts/network.js

```diff
@@ -837,11 +836,14 @@
                                                 dataType: "json",
                                                 async: false,
                                                 success: function (json) {
-                                                    var vpcOfferingObjs = json.listvpcofferingsresponse.vpcoffering;
-                                                    $(vpcOfferingObjs).each(function () {
+                                                    var offerings = json.listvpcofferingsresponse.vpcoffering;
+                                                    var filteredofferings = $.grep(offerings, function (offering) {
+                                                        return offering.state == 'Enabled';
+                                                    });
+                                                    $(filteredofferings).each(function (offering) {
```

Comment:
Can you change this to `filteredofferings.forEach(function (offering) {` as the current line is filling the item list with undefined values.